app/manageEvent/[id]/page.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+
+import { useState, useEffect } from "react";
+import { Event } from "@prisma/client";
+import RouteButton from "@/components/RouteButton";
+
+export default function ManageEvent({ params }: { params: { id: string } }) {
+	const [event, setEvent] = useState<Event>();
+	
+	useEffect(() => {
+		const getEvent = async () => {
+			const id = params.id;
+			const res = await fetch(`/api/event?id=${id}`, { method: 'GET' });
```

Comment:
Should we make this url string a constant in a constants.ts file? That way if it changes there's only one place that will need to be updated. This could also be applied to all the routes, we can have constants so we don't need to change them in multiple places if we decide to change the url of the route.

Reply:
Yea that could be something we do but it seems like it might only be necessary and effective if we were re-using routes.  In the current project scheme, it's unlikely for these routes to re-occur. So idk if I'll add that in this branch, but if we get to a point where we need to re-use routes then I will.